ydb/docs/en/core/yql/reference/yql-core/udf/list/_includes/index/list.md

```diff
@@ -1,13 +1,14 @@
+* [DateTime](../../datetime.md)
+* [Digest](../../digest.md)
+* [Histogram](../../histogram.md)
 * [Hyperscan](../../hyperscan.md)
+* [Ip](../../ip.md)
+* [Knn](../../knn.md)
+* [Math](../../math.md)
```

Comment:
The previous order looks like by descending popularity, and you're changing it to lexicographic order. There might be a discussion on which one is better, but in any case, this change is unrelated to KNN.

Reply:
39ec11af2cfeae6592e34d41edd9a883b846823c